packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -53,6 +54,7 @@ export default class RatingBar extends Component {
           width={itemWidth}
           pathD={pathD}
           viewBox={viewBox}
+          polygonPoints={polygonPoints}
```

Comment:
`polygonPoints` is missing from the `propTypes`. Also, do we really need to enforce all of these props here? Can we spread them instead so we don't have to keep adding support for new props?